contracts/controller/PowerEnabled.sol

```diff
@@ -125,7 +131,7 @@ contract PowerEnabled is MarketEnabled {
   function createDownRequest(address _owner, uint256 _amountPower) public onlyPower whenNotPaused {
     // prevent powering down tiny amounts
     // when powering down, at least totalSupply/minShare Power should be claimed
-    require(_amountPower >= authorizedPower().div(10000)); // minShare = 10000;
+    require(_amountPower >= authorizedPower().div(MIN_SHARE_OF_POWER)); // minShare = 10000;
```

Comment:
not sure we should make min power downs smaller as well @johannbarbie

Reply:
You might want to update the comment on line 134, as it may be confusing.

Reply:
good point!